Cargo.toml

```diff
@@ -15,194 +15,189 @@ members = [
+sp-inherents = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.26" }
+sp-io = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.26" }
+sp-keyring = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.26" }
+sp-keystore = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.26" }
+sp-maybe-compressed-blob = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.26" }
+sp-npos-elections = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.26" }
+sp-offchain = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.26" }
+sp-rpc = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.26" }
+sp-runtime = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.26" }
+sp-runtime-interface = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.26" }
+sp-runtime-interface-proc-macro = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.26" }
+sp-serializer = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.26" }
+sp-session = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.26" }
+sp-staking = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.26" }
+sp-state-machine = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.26" }
+sp-std = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.26" }
+sp-storage = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.26" }
+sp-timestamp = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.26" }
+sp-tracing = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.26" }
+sp-transaction-pool = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.26" }
+sp-trie = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.26" }
+sp-version = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.26" }
+sp-wasm-interface = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.26" }
+substrate-build-script-utils = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.26" }
+substrate-frame-rpc-system = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.26" }
+substrate-prometheus-endpoint = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.26" }
+substrate-wasm-builder = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.26" }
+try-runtime-cli = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.26" }
 
 [patch."https://github.com/paritytech/polkadot"]
-kusama-runtime = { git = "https://github.com/apopiak/polkadot", rev = "5987afea534400f38cd14f699433b9443cf08e42" }
```

Comment:
@Roznovjak this needs to point to the modified version of polkadot by alex, he backported the changes into .25 as well here https://github.com/apopiak/polkadot/tree/apopiak/fix-xcm-executor-atomicity-9-25, you should backport it to the .26 if not compatible

Reply:
Are you sure this doesn't change the relay part of the node or validity of our blocks? Is it just some non-transaction thing patched there or what's up?

Reply:
xcm executor is just implementation of xcm in our runtime, patched to eliminate some cases where assets are usually trapped

Reply:
more info here: https://github.com/galacticcouncil/Basilisk-node/pull/526

Reply:
should be done